MaeParser.cpp

```diff
@@ -351,12 +351,12 @@ std::shared_ptr<Block> MaeParser::blockBody(const std::string& name)
             break;
         case 'b':
             block->setBoolProperty((*(*iter)),
-                                   parse_value<BoolProperty>(m_buffer));
+                                   1u == parse_value<BoolProperty>(m_buffer));
             break;
         }
     }
     schrodinger::mae::whitespace(m_buffer);
-    while (true) {
+    for (;;) {
```

Comment:
ugh, I like the `while(true)` better. I'm trying to think about a tidy way to put the break condition in the loop header...

best I can come up with:

    auto advance = [](&m_buffer) {
        schrodinger::mae::whitespace(m_buffer);
        if (!m_buffer.load()) {
    ￼        throw read_exception(m_buffer, "Missing '}' for block.");
        }
        ++m_buffer.current
    }

    advance()
    for (;*m_buffer.current != '}'; advance()) {

Reply:
I also prefer `while (true)`, but I also prefer the `for (;;)` to your suggestion, it looks way more complicated.